### PR TITLE
Multilanguage support in Dialog Menu

### PR DESCRIPTION
**Pull request by mimizid**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:
    Multilanguage support in Dialog Menu and translation into Russian

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in singleplayer?
2. [x] Have you loaded the mission in LAN host?
3. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### CODE REVIEW

File: A3-Antistasi/Stringtable.xml [hunk 1]
```diff
@@ -2510,5 +2510,737 @@
         <Korean>LanCommi : 이미지 지원&lt;br/&gt;Goon and jw custom: 네이팜 코드 일부 제공&lt;br/&gt;rebel12340:라디오 먹통 스크립트 제공&lt;br/&gt;Manko: 이어플러그 제공&lt;br/&gt;bobchaos: 커뮤니티 관리&lt;br/&gt;그리고 그 외 모든 Github와 스팀 커뮤니티에서 관심을 가져주신 분들.</Korean>
       </Key>
     </Container>
+	<Container name="game_options">
+		<Key ID="STR_antistasi_dialogs_maps_game_options">
```
Comment (CalebSerafin) [inline A3-Antistasi/Stringtable.xml:2514]: As far as I can tell, the work looks solid. 
However, we use spaces to indent lines.<br/>
In the stringtable one indent is equal to 2 spaces (usually 4 in the rest of the code).<br/>
So the container tag will be indented 4 spaces total,
and the Key tag 6 spaces total, etc.
```suggestion
    <Container name="game_options">
      <Key ID="STR_antistasi_dialogs_maps_game_options">
```

Comment (mimizid) [reply A3-Antistasi/Stringtable.xml]: ok, I'll fix it now

**Comment by Bob-Murphy** [comment]
Hi mimizid,
thank you very much for your PRs!
As we currently have a lot of stuff in the pipeline it can take some time to get everything properly reviewed.
For the russian translations, we also will let somebody with russian capabilities check your translations as we have a general 4-eye-principle.
We will certainly use your changes even though we might ask for slight adaptations.
Nevertheless, well done and thank you.

**Review by Bob-Murphy — APPROVED** [review]

**Comment by Bob-Murphy** [comment]
Format is reviewed
Still awaiting review for russian translation

File: A3-Antistasi/Stringtable.xml [hunk 2]
```diff
@@ -2510,5 +2510,737 @@
...
+      </Key>
+      <Key ID="STR_antistasi_dialogs_mission_menu_rescue_missions">
+        <Original>Rescue Missions</Original>
+        <Russian>Спасательные миссии</Russian>
+      </Key>
+    </Container>
+    <Container name="radio_comm">
+      <Key ID="STR_antistasi_dialogs_radio_comm">
+        <Original>Battle Options</Original>
+        <Russian>Радио</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2636]: что за радио? не помню что-то такого

File: A3-Antistasi/Stringtable.xml [hunk 3]
```diff
@@ -2510,5 +2510,737 @@
...
+        <Original>Teleport your squad or a HC selected squad to a friendly zone depending on several factors</Original>
+        <Russian>Телепортируйте свой отряд или отряд, выбранный HC, в дружественную зону в зависимости от нескольких факторов</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_radio_comm_player_money">
+        <Original>Player and Money</Original>
+        <Russian>Игрок и деньги</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_radio_comm_player_money_tooltip">
+        <Original>Look at some player and interact with him</Original>
+        <Russian>Посмотрите на игрока чтобы взаимодействовать</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2652]: Посмотрите на игрока чтобы взаимодействовать с ним

File: A3-Antistasi/Stringtable.xml [hunk 4]
```diff
@@ -2510,5 +2510,737 @@
...
+        <Original>Add to Air Support</Original>
+        <Russian>Добавить в воздушную поддержку</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_vehicle_manager_add_air_support_tooltip">
+        <Original>Gain Airstrike points giving this vehicle to the faction Air pool</Original>
+        <Russian>Получите очки авиаудара, отдав этот автомобиль в запас воздуха фракции</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_vehicle_manager_unlock_vehicle">
+        <Original>Unlock Vehicle</Original>
+        <Russian>Разблокировать автомобиль</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2734]: Разблокировать машину

File: A3-Antistasi/Stringtable.xml [hunk 5]
```diff
@@ -2510,5 +2510,737 @@
...
+      </Key>
+      <Key ID="STR_antistasi_dialogs_vehicle_manager_veh_query">
+        <Original>Add Vehicle to Squad?</Original>
+        <Russian>Добавить машину в отряд?</Russian>
+      </Key>
+    </Container>
+    <Container name="squad_manager">
+      <Key ID="STR_antistasi_dialogs_squad_manager">
+        <Original>HC Squad Options</Original>
+        <Russian>Верховное командование</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2748]: Параметры управления отрядами

File: A3-Antistasi/Stringtable.xml [hunk 6]
```diff
@@ -2510,5 +2510,737 @@
...
+      </Key>
+      <Key ID="STR_antistasi_dialogs_squad_manager_add_vehicle">
+        <Original>Squad Add Vehicle</Original>
+        <Russian>Добавить технику отряду</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2752]: Добавить технику в отряд

File: A3-Antistasi/Stringtable.xml [hunk 7]
```diff
@@ -2510,5 +2510,737 @@
...
+      </Key>
+      <Key ID="STR_antistasi_dialogs_squad_manager_add_vehicle_tooltip">
+        <Original>Look at some vehicle and assign it to the selected squad for it's use</Original>
+        <Russian>Посмотрите на какую-нибудь технику и назначьте ее выбранному отряду для использования</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_squad_manager_squad_sitrep">
+        <Original>Squad SITREP</Original>
+        <Russian>Состояние отряда</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2760]: Статус отряда

File: A3-Antistasi/Stringtable.xml [hunk 8]
```diff
@@ -2510,5 +2510,737 @@
...
+        <Original>Mount / Dismount</Original>
+        <Russian>Погрузка / Разгрузка</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_squad_manager_mount_tooltip">
+        <Original>Force squad to mount or dismount their assigned vehicle</Original>
+        <Russian>Заставить отряд сесть или слезть с назначенной машины</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_squad_manager_static_autotarget">
+        <Original>Static Autotarget</Original>
+        <Russian>Статическая автоцель</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2776]: Автонаведение установок

File: A3-Antistasi/Stringtable.xml [hunk 9]
```diff
@@ -2510,5 +2510,737 @@
...
+        <Original>Auto Rearm \ Loot</Original>
+        <Russian>Автоперезарядка \ Лут</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_AI_management_auto_rearm_tooltip">
+        <Original>AI will search for better weapons, vests, helmets etc. If they are in a vehicle, they will just store what they scavenge in the vehicle. If not, they will equip them</Original>
+        <Russian>ИИ будет искать лучшее оружие, жилетки, шлемы и т. Д. Если они находятся в транспортном средстве, они просто сохранят то, что они собирают, в транспортном средстве. Если нет, они их оснастят</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_AI_management_auto_heal">
+        <Original>Auto Heal</Original>
+        <Russian>Авто исцеление</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2806]: Авто-лечение

File: A3-Antistasi/Stringtable.xml [hunk 10]
```diff
@@ -2510,5 +2510,737 @@
...
+      </Key>
+      <Key ID="STR_antistasi_dialogs_AI_management_auto_heal_tooltip">
+        <Original>AI squad mates will heal proactively each other</Original>
+        <Russian>Соратники ИИ будут активно лечить друг друга</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_AI_management_SITREP">
+        <Original>Squad SITREP</Original>
+        <Russian>Состояние отряда</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2814]: Статус отряда

File: A3-Antistasi/Stringtable.xml [hunk 11]
```diff
@@ -2510,5 +2510,737 @@
...
+        <Original>Garrison Units / Squads</Original>
+        <Russian>Добавить в гарнизон</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_AI_management_garrison_tooltip">
+        <Original>Adds selected units or squads to a map selected garrison</Original>
+        <Russian>Добавляет выбранные юниты или отряды в выбранный на карте гарнизон</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_AI_management_squad_add_vehicle">
+        <Original>Squad Add Vehicle</Original>
+        <Russian>Добавить технику отряду</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2830]: Добавить технику в отряд

File: A3-Antistasi/Stringtable.xml [hunk 12]
```diff
@@ -2510,5 +2510,737 @@
...
+      </Key>
+      <Key ID="STR_antistasi_dialogs_AI_management_squad_add_vehicle_tooltip">
+        <Original>Look at some vehicle and assign it to the selected squad for it's use</Original>
+        <Russian>Посмотрите на какую-нибудь технику и назначьте ее выбранному отряду для использования</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_AI_management_dismiss_units">
+        <Original>Dismiss Units / Squad</Original>
+        <Russian>Расформировать</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2838]: Роспуск

File: A3-Antistasi/Stringtable.xml [hunk 13]
```diff
@@ -2510,5 +2510,737 @@
...
+        <Original>Recruit Squad</Original>
+        <Russian>Набрать отряд</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_commander_comm_recruit_tooltip">
+        <Original>Recruit new squads and manage them with the HC Module (CTRL + SPACE)</Original>
+        <Russian>Набирайте новые отряды и управляйте ими с помощью модуля HC (CTRL + SPACE)</Russian>
+      </Key>
+      <Key ID="STR_antistasi_dialogs_commander_comm_air_support">
+        <Original>Air Support</Original>
+        <Russian>Воздушная поддержка</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2868]: Авиаподдержка

File: A3-Antistasi/Stringtable.xml [hunk 14]
```diff
@@ -2510,5 +2510,737 @@
...
+      </Key>
+      <Key ID="STR_antistasi_dialogs_commander_comm_air_support_tooltip">
+        <Original>Ask for Air Support (uses Airstrike points)</Original>
+        <Russian>Попросите воздушную поддержку (использует очки авиаудара)</Russian>
```
Comment (AlexTriada) [inline A3-Antistasi/Stringtable.xml:2872]: Запросить авиаподдержку (использует очки авиаподдержки)

**Comment by AlexTriada** [comment]
in general, not bad, you can leave it as it is

**Comment by Bob-Murphy** [comment]
@mimizid 
Please check the comments made by Alex and resolve them.
Thank you.

File: A3-Antistasi/Stringtable.xml (hunk 12 above)
Comment (AlexTriada) [reply A3-Antistasi/Stringtable.xml:2838]: В запас

File: A3-Antistasi/Stringtable.xml (hunk 2 above)
Comment (mimizid) [reply A3-Antistasi/Stringtable.xml:2636]: Диалог называется radio_comm, вот и написал так. Не называть же "настройки боя" или "управление матчем"...хмм...хотя... можно просто "Управление"

File: A3-Antistasi/Stringtable.xml (hunk 5 above)
Comment (mimizid) [reply A3-Antistasi/Stringtable.xml:2748]: Слишком длинно, наверное, будет. Можно просто "Управление отрядами"

Comment (AlexTriada) [reply A3-Antistasi/Stringtable.xml]: отличный выбор

**Comment by AlexTriada** [comment]
translate is good

**Comment by Bob-Murphy** [comment]
Thank you. Will look at our on Monday.

**Review by Bob-Murphy — APPROVED** [review]

**Pull request merged**